utils/forest_shim/src/state_tree.rs

```diff
@@ -331,3 +331,13 @@ impl From<&ActorState> for ActorStateV2 {
         }
     }
 }
+
+impl quickcheck::Arbitrary for ActorState {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        let cid = Cid::new_v1(
```

Comment:
How abt `Cid::arbitrary(g)`, I believe it's available under `arb` feature. See https://docs.rs/crate/cid/0.8.6/features

Reply:
It's the wrong version of `quickcheck`, unfortunately. Version `0.9` vs `1.0`. We submitted patches to fix that, but it's only available in later CID versions. And we cannot upgrade the CID crate because of the FVM.